### ISSUE TITLE
README Cleanup

### ISSUE THREAD

**Issue by gheskett** [HackerN64/HackerSM64#846]
The README is currently very outdated and is a nightmare to sift through. It contains swaths of info about patches/defines (that are now too plentiful to exist in a README) and lacks mention of things that actually should belong in a README (like linking to the HackerSM64 installation Wiki). The legacy UltraSM64 README also bloats it a fair amount with very little added value at this point.

There is some subjectivity to this, such as how we'd want to approach cleaning up credits and where to put information about changes to the base game (especially ones not covered by defines). We'd also need to pool together some effort on that front to mention important changes that currently aren't present in the existing README.

**Comment by LuigiXHero** [comment]
Yeah please fix this, I just spent over an hour of my life trying to figure out how to build this shit from the readme instructions only to find it's on the wiki page that isn't mentioned anywhere after finally asking someone. I'm too anti-social for this. Also you can split it up into multiple md files and link between them you don't need to cram everything into one file.
